integration-tests/deployment/ccip/deploy_home_chain.go

```diff
@@ -52,6 +52,21 @@ const (
 	MaxDurationShouldTransmitAcceptedReport = 10 * time.Second
 )
 
+var (
+	// TODO: sort out why the mismatch here
+	//CCIPCapabilityId [32]byte = utils.MustHash(hexutil.Encode(MustABIEncode(`[{"type": "string"}, {"type": "string"}]`, CapabilityLabelledName, CapabilityVersion)))
+	CCIPCapabilityId = MustHashFromBytes(hexutil.MustDecode("0xe0da3c2b9005178f4731c9f40164f1933ad00bac9d6c13ad4ca1a8a763416380"))
```

Comment:
This just avoids RPC calls all the time to get the capability ID. We expect it to be fixed per version